Apps/W1/HybridGP/app/src/Migration/Support/PurchaseOrders/GPPOPHeader.table.al

```diff
@@ -797,6 +798,9 @@ table 40102 "GP POPPOHeader"
         PurchaseDocumentStatus: Enum "Purchase Document Status";
         CountryCode: Code[10];
     begin
+        if not GPCompanyAdditionalSettings.GetMigrateOpenPOs() then
```

Comment:
is this needed? You have the same check in the helper functions which calls this procedure.